docs/DeveloperGuide.md

```diff
@@ -295,28 +295,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `iVolunteer` and the **Actor** is the `Volunteer Coordinator`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case 2: List all volunteering events**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all volunteering events.
+2.  iVolunteer list out a summarized version of all volunteering events.
 
     Use case ends.
 
 **Extensions**
 
+* 1a. The input command is incorrect.
+
+    * 1a1. System prompts the user to provide a valid command.
+      
+      Use case resumes from step 1.
+
 * 2a. The list is empty.
```

Comment:
Correct me if I'm wrong but this should be `1b`

Reply:
`1a` is fine. this is the first extension for 1

Reply:
I meant that `2a` shd be `1b` instead cos `The list is empty` happens after step 1, not after step 2. Thoughts?

Reply:
hmm in this case i agree, though i would put the extension as `There are no events to list`

Reply:
yup makes sense @ruo-x

Reply:
agreee

Reply:
i dunno i followed the AB3 original one and they put it at 2a tho

Reply:
coz 1b makes sense too ill j change it